src/main/java/com/dkohut/chatservice/common/interfaces/IChatRoomDAOService.java

```diff
@@ -0,0 +1,55 @@
+package com.dkohut.chatservice.common.interfaces;
+
+import java.util.List;
+
+import com.dkohut.chatservice.common.entity.ChatRoom;
+
+/**
+ * 
+ * @author Dmytro Kohut
+ *
+ */
+public interface IChatRoomDAOService {
+
+	/**
+	 * Select chat rooms in given range, sorted by created_at column.
+	 * 
+	 * @param Integer firstRow
+	 * @param Integer numberOfRows 
+	 * @return List<ChatRoom>
+	 */
+	List<ChatRoom> selectInRange(Integer firstRow, Integer numberOfRows);
```

Comment:
bad name of method. Not clear why we need this method at all

Reply:
Method name was changed.